packages/history/src/models/migrations/20181218111220-create-index-WorkflowTransHist_boid_idx.js

```diff
@@ -3,9 +3,12 @@
 const indexName = 'WorkflowTransHist_boid_idx';
 
 module.exports = {
-  up: (queryInterface, DataTypes) => queryInterface.addIndex('WorkflowTransitionHistory', {
-    fields: ['businessObjId'],
-    name: indexName
-  }),
+  up: (queryInterface, DataTypes) => queryInterface.addIndex(
+    'WorkflowTransitionHistory',
+    ['businessObjId'],
```

Comment:
Please revert this change, because according to https://sequelize.org/master/class/lib/dialects/abstract/query-interface.js~QueryInterface.html#instance-method-addIndex previous definition is recommended: 
![Screenshot 2021-09-22 at 13 25 20](https://user-images.githubusercontent.com/31243790/134327382-46267a38-dc15-438e-835d-3b7101b91744.png)